uktrade/enquiry-mgmt-tool
# Update specific investment programme payload field

## Description of change

Due to recent changes to the `specific_programme` field in Data Hub API (converted from foreign key to many-to-many), the field requirements in the payload that EMT sends to DataHub when creating an investment project have changed.

This PR applies these changes to ensure that created investment projects will record the specific_investment_programme data.

- Adds an 's' to the field name
- Ensures the payload value is a list of ID's, instead of a single ID

## Test instructions

All current tests should pass. 

If you create an enquiry in EMT, the specific investment programme should now be visible in the created Data Hub investment project.

- You will need to create an enquiry using the csv template provided on the ‘Import enquiries’ page
- Submit the enquiry to DH
- On DH - Staging, review the created investment project and check that the Specific Investment Programme data has successfully been added to the new project

Although, maybe we need to further test the interaction between EMT and DataHub to ensure breaking changes like this don't go under the radar.
